index.js

```diff
@@ -0,0 +1,217 @@
+//. # Sanctuary Benchmark
+//.
+//. Small wrapper over [benchmarkjs][1] to enable a consistent style of
+//. benchmarking across all Sanctuary projects. Allows for the easy definition
+//. of comparative benchmarks, and outputs results in a standardized format for
+//. sharing.
```

Comment:
Let's begin this paragraph with a complete sentence:

_S<ins>anctuary Benchmark is a s</ins>mall wrapper…_